meet/start-service.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+ADDITIONAL_KOPANO_PACKAGES=${ADDITIONAL_KOPANO_PACKAGES:-""}
+
+set -eu # unset variables are errors & non-zero return values exit the whole script
+
+if [ ! -e /kopano/$SERVICE_TO_START.py ]; then
+	echo "Invalid service specified: $SERVICE_TO_START" | ts
+	exit 1
+fi
+
+echo "Configure core service '$SERVICE_TO_START'" | ts
```

Comment:
What does "core service" mean?

Reply:
that line was a copy and paste from the "core" docker image. I was a bit torn between just adding the meet relevant stuff on top, or making a "new" image without all the core components inside.